benchmarks/module/conv_model.py

```diff
@@ -68,10 +74,23 @@ def run_conv_model(get_clients: Callable[[int], List[Any]]):
 
     model.eval()
 
-    # For the moment we have only inference
     expected = model(x_secret)
 
-    res_mpc = mpc_model(x_mpc)
+    return (expected, mpc_model, x_mpc)
+
+
+def run_inference_conv_model(
+    expected: sy.Module, mpc_model: sy.Module, x_mpc: MPCTensor
+):
+    """Run the inference on the passed in model.
 
+    Args:
+        expected: the origin model
+        mpc_model: the Syft model
+        x_mpc: the MPCTensor
+    """
+    """Run the inference on the passed in model"""
+    res_mpc = mpc_model(x_mpc)
     res_mpc.reconstruct()
+
     expected = expected.detach().numpy()
```

Comment:
you can remove these, we don't need them 😃

Reply:
@aanurraj is there any todo for us to merge this PR?

Reply:
I think it is good to go! I will ping @aanurraj